CMakeLists.txt

```diff
@@ -458,7 +458,7 @@ if(ENABLE_MASON)
 
 else()
 
-  find_package(Boost 1.49.0 REQUIRED COMPONENTS ${BOOST_COMPONENTS})
+  find_package(Boost 1.54 REQUIRED COMPONENTS ${BOOST_COMPONENTS})
```

Comment:
Updated the wiki page with cmake 3.7.0 version and added a check to `CMakeLists.txt`